pallets/creditcoin/src/lib.rs

```diff
@@ -1351,5 +1363,40 @@ pub mod pallet {
 
 			Ok(PostDispatchInfo { actual_weight: None, pays_fee: Pays::No })
 		}
+
+		#[transactional]
+		#[pallet::call_index(23)]
+		#[pallet::weight(<T as Config>::WeightInfo::request_burn_gate())]
+		pub fn request_burn_gate(
```

Comment:
This is the function that is causing the issues. The errors are related to the macros for the weights and call index. Do we need to generate the weight info as part of the build process?